rclcpp/QUALITY_DECLARATION.md

```diff
@@ -82,13 +82,13 @@ The API is publicly available in the [ROS 2 API documentation](http://docs.ros2.
 
 The license for `rclcpp` is Apache 2.0, and a summary is in each source file, the type is declared in the [`package.xml`](./package.xml) manifest file, and a full copy of the license is in the [`LICENSE`](../LICENSE) file.
 
-There is an automated test which runs a linter that ensures each file has a license statement. [Here](http://build.ros2.org/view/Epr/job/Epr__rclcpp__ubuntu_bionic_amd64/lastBuild/testReport/rclcpp/) can be found a list with the latest results of the various linters being run on the package.
+There is an automated test which runs a linter that ensures each file has a license statement. [Here](http://build.ros2.org/view/Fpr/job/Fpr__rclcpp__ubuntu_focal_amd64/lastStableBuild/testReport/rclcpp/) can be found a list with the latest results of the various linters being run on the package.
```

Comment:
Just a heads up, lastStableBuild might be a bit old if there are any tests which regularly fail. I know everything is green now, but you may just want to go with `lastSuccessfulBuild`